starboard/testing/fake_graphics_context_provider.cc

```diff
@@ -181,9 +198,14 @@ void FakeGraphicsContextProvider::InitializeEGL() {
 #if BUILDFLAG(IS_ANDROID)
   display_ = EGL_CALL_SIMPLE(eglGetDisplay(EGL_DEFAULT_DISPLAY));
 #else
-  display_ = EGL_CALL_SIMPLE(eglGetPlatformDisplay(
+  // Attempt to use eglGetPlatformDisplayEXT for EGL 1.4.
```

Comment:
I think this is a bit inaccurate: we're trying to import the 
`eglGetPlatformDisplayEXT` symbol from the egl library "by 
hand", and this is because we cannot/don't want to enforce
full eGL 1.5 compat, relying instead on the extension mechanism.